posthog/queries/trends/total_volume.py

```diff
@@ -72,12 +80,23 @@ def _total_volume_query(self, entity: Entity, filter: Filter, team: Team) -> Tup
                     aggregator="distinct_id" if team.aggregate_users_by_distinct_id else "person_id",
                     **trend_event_query.active_user_params,
                 )
-            elif filter.display == TRENDS_CUMULATIVE and entity.math == "dau":
+            elif filter.display == TRENDS_CUMULATIVE and entity.math == UNIQUE_USERS:
```

Comment:
Looks like there's no support for non-time series aggregation? Does the frontend take care of that?